docs/setup_own_server.md

```diff
@@ -83,7 +83,7 @@ Your CouchDB has been initialised successfully. If you want this manually, pleas
 Whatever solutions we can use. For the simplicity, following sample uses Cloudflare Zero Trust for testing.
 
 ```
-$ cloudflared tunnel --url http://localhost:5984
+cloudflared tunnel --url http://localhost:5984
```

Comment:
Would you mind if I ask you to split this code block like one in "Generate the setup URI on a desktop device or server"?

Reply:
sure thing, sorry for taking so long but let me do this now